service/src/main/java/com/fecklessweasel/service/datatier/FileMetadataTable.java

```diff
@@ -0,0 +1,209 @@
+
+        CodeContract.assertNotNull(connection, "connection");
+
+        try{
+          PreparedStatement lookUpFileStatement = connection.prepareStatement(LOOKUP_FILE_QUERY);
+          lookUpFileStatement.setInt(1, fileId);
+
+          return lookUpFileStatement.executeQuery();
+        } catch (SQLException ex) {
+            throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
+        }
+    }
+    /**
+     * Looks up all files belonging to a certain user
+     *@param connection Connection to the mySQL database
+     *@param user: The username whose files we want to return
+     *@return files A ResultSet containing all file tuples associated with the username
+     */
+    public static ResultSet lookUpUsersFiles(Connection connection, String user)
+      throws ServiceException{
+
+        CodeContract.assertNotNull(connection, "connection");
+        CodeContract.assertNotNullOrEmptyOrWhitespace(user, "user");
+
+        try{
+          PreparedStatement lookUpFileStatement = connection.prepareStatement(LOOKUP_FILES_FROM_USER_QUERY);
+          lookUpFileStatement.setString(1, user);
+
+          return lookUpFileStatement.executeQuery();
+        } catch (SQLException ex) {
+            throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
+        }
+
+    }
+
+    /**
+     * Looks up all files belonging to a certain user
+     *@param connection Connection to the mySQL database
+     *@param course The course we want files from
+     *@param university The university that the course belongs to
+     *@result files A ResultSet containing all file tuples associated with the username
+     */
+    public static ResultSet lookUpCourseFiles(Connection connection, String course, String university)
+      throws ServiceException{
+
+              CodeContract.assertNotNull(connection, "connection");
+              CodeContract.assertNotNullOrEmptyOrWhitespace(course, "course");
+              CodeContract.assertNotNullOrEmptyOrWhitespace(university, "university");
+
+              try{
+                PreparedStatement lookUpFileStatement = connection.prepareStatement(LOOKUP_FILES_FROM_USER_QUERY);
+                lookUpFileStatement.setString(1, course);
+                lookUpFileStatement.setString(1, university);
+
+                return lookUpFileStatement.executeQuery();
+              } catch (SQLException ex) {
+                  throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
+              }
+    }
+    /**
+     * Deletes the specified file's metadata from the table
+     *@param connection Connection to the mySQL database
+     *@param fid The file's unique identifier
+     */
+    public static void deleteFile(Connection connection, int fid) throws ServiceException{
+      try {
+        PreparedStatement deleteStatement = connection.prepareStatement(DELETE_FILE_QUERY);
+        deleteStatement.setInt(1, fid);
+
+        if (deleteStatement.executeUpdate() != -1) {
```

Comment:
This should be 1, not -1. executeUpdate() returns the number of rows that were updated.